supplements/tomb-of-annihilation/items.xml

```diff
@@ -0,0 +1,263 @@
+			<p class="indent"><b><i>Sentience.</i></b> The <em>Ring of Winter</em> is a sentient chaotic evil item with an Intelligence of 14, a Wisdom of 14, and a Charisma of 17. The ring communicates by transmitting emotion to the creature carrying or wielding it, and it has hearing and normal vision out to a range of 60 feet. The ring craves destruction, and it likes inflicting indiscriminate harm on others.</p>
+			<p class="indent"><b><i>Nondetection.</i></b> The <em>Ring of Winter</em> defies attempts to magically locate it. Neither the ring nor its wearer can be targeted by any divination magic or perceived through magical scrying sensors.</p>
+			<p class="indent"><b><i>Frozen Time.</i></b> As long as you wear the ring, you don't age naturally. This effect is similar to suspended animation, in that your age doesn't catch up to you once the ring is removed. The ring doesn't protect its wearer from magical or supernatural aging effects, such as the Horrifying Visage of a ghost.</p>
+			<p class="indent"><b><i>Cold Immunity.</i></b> While attuned to and wearing the ring, you have immunity to cold damage and don't suffer any ill effects from extreme cold (see chapter 5 of the <i>Dungeon Master's Guide</i>).</p>
+			<p class="indent"><b><i>Magic.</i></b> The <em>Ring of Winter</em> has 12 charges and regains all its expended charges daily at dawn. While wearing the ring, you can expend the necessary number of charges to activate one of the following properties:</p>
+			<ul>
+				<li>You can expend 1 charge as an action and use the ring to lower the temperature in a 120-foot-radius sphere centered on a point you can see within 300 feet of you. The temperature in that area drops 20 degrees per minute, to a minimum of -30 degrees Fahrenheit. Frost and ice begin to form on surfaces once the temperature drops below 32 degrees. This effect is permanent unless you use the ring to end it as an action, at which point the temperature in the area returns to normal at a rate of 10 degrees per minute.</li>
+				<li>You can cast one of the following spells from the ring (spell save DC 17) by expending the necessary number of charges: <em>Bigby's hand</em> (2 charges; the hand is made of ice, is immune to cold damage, and deals bludgeoning damage instead of force damage as a clenched fist), <em>cone of cold</em> (2 charges), <em>flesh to ice</em> (3 charges; as <i>flesh to stone<i> except that the target turns to solid ice with the density and durability of stone), <em>ice storm</em> (2 charges), <em>Otiluke's freezing sphere</em> (3 charges), <em>sleet storm</em> (1 charge), <em>spike growth</em> (1 charge; the spikes are made of ice), or <em>wall of ice</em> (2 charges).</li>
```

Comment:
```suggestion
				<li>You can cast one of the following spells from the ring (spell save DC 17) by expending the necessary number of charges: <em>Bigby's hand</em> (2 charges; the hand is made of ice, is immune to cold damage, and deals bludgeoning damage instead of force damage as a clenched fist), <em>cone of cold</em> (2 charges), <em>flesh to ice</em> (3 charges; as <i>flesh to stone</i> except that the target turns to solid ice with the density and durability of stone), <em>ice storm</em> (2 charges), <em>Otiluke's freezing sphere</em> (3 charges), <em>sleet storm</em> (1 charge), <em>spike growth</em> (1 charge; the spikes are made of ice), or <em>wall of ice</em> (2 charges).</li>
```